android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java

```diff
@@ -191,6 +191,16 @@ public void trackSocialInteraction(String trackerId, String network, String acti
     }
 
     @ReactMethod
+    public void setCustomDimensionValue(String trackerId, Integer dimensionIndex, String dimensionValue)
+    {
+        Tracker tracker = getTracker(trackerId);
+
+        if (tracker != null) {
+          tracker.set(Fields.customDimension(dimensionIndex), dimensionValue);
```

Comment:
Yes, the latest iOS GA lib (which is what is linked to) is v3, the latest Android is v4.
Sorry for the confusion.